msm-sitemap.php

```diff
@@ -597,21 +597,33 @@ public static function create_post_type() {
 	 * @return object[] modified posts
 	 */
 	public static function get_last_modified_posts() {
-		global $wpdb;
+        global $wpdb;
+        
+        $sitemap_last_run = get_option( 'msm_sitemap_update_last_run', false );
 
-		$sitemap_last_run = get_option( 'msm_sitemap_update_last_run', false );
+        $date = date( 'Y-m-d H:i:s', ( current_time( 'timestamp', 1 ) - 3600 ) ); // posts changed within the last hour
 
-		$date = date( 'Y-m-d H:i:s', ( current_time( 'timestamp', 1 ) - 3600 ) ); // posts changed within the last hour
+        if ( $sitemap_last_run ) {
+            $date = date( 'Y-m-d H:i:s', $sitemap_last_run );
+        }
 
-		if ( $sitemap_last_run ) {
-			$date = date( 'Y-m-d H:i:s', $sitemap_last_run );
-		}
+        $post_types_in = self::get_supported_post_types_in();
+                $count_posts = $wpdb->get_var( "SELECT count(*) as total FROM $wpdb->posts WHERE post_type IN ( {$post_types_in} ) AND post_modified_gmt >= '".$date."' ");
```

Comment:
`$wpdb->prepare` should be used to sanitize the query first.

Reply:
@nathanielks  We made the requested changes with new commit. Please review once.
Thanks.